test/qunit-pjv.js

```diff
@@ -62,7 +62,8 @@ QUnit.test("Dependencies Ranges", function() {
             'tilde-first': '~1.2',
             'x-version': '1.2.x',
             'tilde-top': '~1',
-            'caret-top': '^1'
+            'caret-top': '^1',
+            "@es-shims/api": "^1.0.0"
```

Comment:
It would have been nice to follow the existing style of single quotes here...

Reply:
```suggestion
            '@es-shims/api': '^1.0.0'
```